app/search_builders/hyrax/nested_collections_parent_search_builder.rb

```diff
@@ -22,7 +22,7 @@ def initialize(scope:, child:, page:)
     # @return [void]
     def parent_collections_only(solr_parameters)
       solr_parameters[:fq] ||= []
-      solr_parameters[:fq] += [Hyrax::SolrQueryBuilderService.construct_query_for_ids(child.member_of_collection_ids)]
+      solr_parameters[:fq] += [Hyrax::SolrQueryService.new.with_ids(ids: child.member_of_collection_ids).build]
```

Comment:
I think this one is not quite right. Many specs are failing here.

Reply:
Looking at it now.

Reply:
Just added to it with [a9d4d7f](https://github.com/samvera/hyrax/pull/5608/commits/a9d4d7f30c55525104625de0e60546031b43f2db) to address that issue.